src/Arcus.Observability.Tests.Runtimes.AzureFunction/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM mcr.microsoft.com/azure-functions/dotnet:4.0 AS base
 WORKDIR /home/site/wwwroot
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/sdk:6.0.100-bullseye-slim AS build
+FROM mcr.microsoft.com/dotnet/sdk:8.0-preview-bullseye-slim AS build
```

Comment:
If I'm not mistaken, this container is used to run the tests inside a container.
is there a possibility to run these tests for multiple .NET targets ?

Like, maybe parametrize/tokenize the base-images in the dockerfile, and build the container multiple times in the pipeline, using different base images ?

Reply:
Yes, that could be possible, maybe with a YAML loop for base images and multiple test suite runs for the Docker-specific integration tests.
Though, not sure if this should be a blocking factor here as this is probably something that we will be using across multiple repo's (plus, my Docker skills are very very poor 😄).